backend/app/core/mail.py

```
@@ -33,20 +33,28 @@ async def get_user_email(user_id: int, db: AsyncSession) -> Optional[str]:
     except Exception as e:
         raise DatabaseError(f"Failed to retrieve user email: {str(e)}")
 
-async def send_email(email_data: EmailSchema) -> None:
+# CHANGED: Update send_email function signature to accept individual parameters instead of EmailSchema
+async def send_email(
+    to_email: str,
+    subject: str, 
+    body: str,
+    cc_emails: Optional[List[str]] = None,
+    bcc_emails: Optional[List[str]] = None,
+    request_id: Optional[str] = None
```

Ruff (0.12.2)

Unused function argument: `request_id`

(ARG001)